Dockerfile

```diff
@@ -64,6 +39,34 @@ COPY ./frontend ./
 RUN npm run build
 # All the built frontend files for the SPA are now in '/app/build/'
 
+############################################################
+# Backend Build
+############################################################
+FROM golang:1.18-alpine as builder
+
+ARG BUILD_TIMESTAMP
+ARG VERSION
+
+RUN apk update && apk add --no-cache git ca-certificates && update-ca-certificates
+
+WORKDIR /app
+
+COPY ./backend/go.mod .
+COPY ./backend/go.sum .
+RUN go mod download
+
+COPY ./backend .
+# Copy frontend build into embed directory so that we can embed
+# the SPA into the Go binary via go embed.
+COPY --from=frontendBuilder /app/build/ ./pkg/embed/frontend
+
+RUN CGO_ENABLED=0 go build \
+-ldflags="-w -s \
+    -X github.com/redpanda-data/console/backend/pkg/version.Version=$VERSION \
+    -X github.com/redpanda-data/console/backend/pkg/version.BuiltAt=$BUILD_TIMESTAMP" \
```

Comment:
Time stamps on builds prevent reproducibility

using git version should be enough